shipping_locales.txt

```diff
@@ -0,0 +1,73 @@
+af
```

Comment:
This file is used to [import the available](https://github.com/mozilla-mobile/firefox-ios/blob/3b9c2d558f08e80d13b073c5433c1e0718c00460/bitrise.yml#L66) locales and run the tests with each one